crates/sui-graphql-rpc/src/server/builder.rs

```diff
@@ -317,7 +317,9 @@ impl ServerBuilder {
             builder = builder.extension(QueryLimitsChecker::default());
         }
         if config.internal_features.query_timeout {
-            builder = builder.extension(Timeout);
+            builder = builder.extension(Timeout {
```

Comment:
Please create a constructor function for the timeout -- this is leaking an implementation detail at the moment.